Makefile

```diff
@@ -12,6 +12,7 @@ MINIKUBE_DISKSIZE ?= 40g
 MINIKUBE_DRIVER ?= kvm2
 MINIKUBE_MEMORY ?= 3072
 MINIKUBE_PROFILE ?= minikube
+MINIKUBE_KUBERNETES ?= v1.15.4
```

Comment:
Please add also the doc line, going like:

```
@echo '  MINIKUBE_KUBERNETES Which Kubernetes version to use with Minikube? [default=v1.15.4]'
```

so that the option would be documented when people type naked `make`.